setup.py

```diff
@@ -2,15 +2,15 @@
 from platform import python_version
 from setuptools import setup
 
-if python_version() < '3.5':
-    raise ImportError("This module requires Python >=3.5")
+if python_version() < '3.6':
+    raise ImportError("This module requires Python >=3.6")
 
 with open('README.md', 'r') as in_file:
     long_description = in_file.read()
 
 setup(
     name='clickplc',
-    version='0.2.6',
+    version='0.2.7',
```

Comment:
I know we're not really using semantic version as intended (I've always justified as anything < 1.0.0 is free reign) but I think python compatibility changes should be at least a minor version bump. Could also wait until after the tags are added.